src/Compilers/CSharp/Test/Syntax/Syntax/SyntaxNormalizerTests.cs

```diff
@@ -16,6 +16,142 @@ namespace Microsoft.CodeAnalysis.CSharp.UnitTests
 {
     public class SyntaxNormalizerTests
     {
+        [Fact, WorkItem(52543, "https://github.com/dotnet/roslyn/issues/52543")]
+        public void TestNormalizePatternInIf()
+        {
+            TestNormalizeStatement(
+                @"{object x = 1;
+                if (x is {})
+                {
+                }
+                if (x is {} t)
+                {
+                }
+                if (x is int {} t2)
+                {
+                }
+                if (x is System.ValueTuple<int, int>(_, _) { Item1: > 10 } t3)
+                {
+                }
+                if (x is System.ValueTuple<int, int>(_, _) { Item1: > 10, Item2: < 20 })
+                {
+                }
+}",
+                @"{
+  object x = 1;
+  if (x is { })
+  {
+  }
+
+  if (x is { } t)
+  {
+  }
+
+  if (x is int { } t2)
+  {
+  }
+
+  if (x is System.ValueTuple<int, int> (_, _) { Item1: > 10 } t3)
+  {
+  }
+
+  if (x is System.ValueTuple<int, int> (_, _) { Item1: > 10, Item2: < 20 })
+  {
+  }
+}"
```

Comment:
Can you try calling [NormalizeLineEndings](https://github.com/dotnet/roslyn/blob/315c2e149ba7889b0937d872274c33fcbfe9af5f/src/Compilers/Test/Core/FX/StringExtensions.cs#L17-L25) on the expected output for failing tests?

```suggestion
}".NormalizeLineEndings()
```

Reply:
I added NormalizeLineEndings() but should work without it...